manageiq-operator/pkg/helpers/miq-components/kafka.go

```diff
@@ -174,9 +174,14 @@ func KafkaDeployment(cr *miqv1alpha1.ManageIQ, scheme *runtime.Scheme) (*appsv1.
 		"app":  cr.Spec.AppName,
 	}
 
+	kafkaImage := cr.Spec.KafkaImage
+	if kafkaImage == "" {
```

Comment:
Same here

Reply:
As an example, you mean have cr.go define cr.Spec.KafkaImage if not specified and built it based on the image parts ?  i.e. cr.Spec.KafkaImage = cr.Spec.KafkaImageName + ":" + cr.Spec.KafkaImageTag ?   So code here below would just use cr.Spec.KafkaImage (now guaranteed to be defined) ?

Reply:
exactly